### DISCUSSION TITLE
Preserve comments while generating entities with sea-orm-cli

### DISCUSSION THREAD

**Discussion by alibektas** [SeaQL/sea-orm#1927] [Q&A]
How can I comment a table which is to be created by a migration I am applying and make those comments appear with the generated entities?
If it is not possible I want to know how you comment your entities in general keeping in mind that you `fresh` your migrations in the course of development so commenting a generated file is not option.

**Comment by tyt2y3** [comment]
That's an interesting idea. I think support of table / column "comment" isn't here, and IIRC SeaQuery does not actually add comments. I think SeaSchema does retrieve the schema comments, but currently aren't using them.

**Reply by tyt2y3** [reply]
I am not sure I'd have the time to work on it, but PR is welcome.

**Reply by alibektas** [reply]
Oh yeah sure that's what I meant too.
